valuable/Cargo.toml

```diff
@@ -37,6 +37,10 @@ valuable-derive = { version = "0.1.0", optional = true, path = "../valuable-deri
 
 [dev-dependencies]
 criterion = "0.3"
+# Plotters increased their MSRV with v0.3.2 to be Rust 1.56
+# We pull in this dependency via criterion.
+# See https://github.com/plotters-rs/plotters/issues/398
+plotters = "= 0.3.1"
```

Comment:
i'm somewhat surprised this is necessary, since the CI MSRV check looks like it isn't checking tests/benchmarks? https://github.com/tokio-rs/valuable/blob/aaa9aa14818c30d00df5a2939303054bc54702ff/.github/workflows/ci.yml#L36

dev-dependencies should be excluded from the MSRV checks --- I don't think we care if users on older Rust versions can _develop_ valuable (e.g. run its benchmarks and tests); we only care if users on MSRV can _depend_ on the crate. if we're currently checking the benchmarks on MSRV on CI, i would prefer to change that, rather than adding an `=` dependency.

cc @carllerche, what do you think?

Reply:
also, i have a minor preference for moving this change to a separate branch, and landing it first; logically, the diff for adding a `PartialEq` impl _shouldn't_ touch the dependencies...